### PR TITLE
Add support to install linux OS using vm_mac paramter

### PR DESCRIPTION
**Pull request by spoorthys1303**
Installation of Linux OS required changes in lpar_netboot command with addition to input paramter vm_mac from user.
When user inputs the vm_mac paramter, linux os installtion starts with the the help of lpar_netboot command with the additional options and vm_mac.

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@AnilVijayan can you please review this pull request

File: plugins/module_utils/hmc_resource.py [hunk 1]
```diff
@@ -491,6 +491,24 @@ def installOSFromNIM(self, loc_code, nimIP, gateway, lparIP, vlanID, vlanPrio, s
             " " + viosName + " " + profName + " " + systemName
         self.hmcconn.execute(lpar_netboot)
 
+    def installlinuxOSFromNIM(self, nimIP, gateway, lparIP, submask, viosName, profName, systemName, lparMac):
```
Comment (naveenkandakur) [inline plugins/module_utils/hmc_resource.py]: Don't Introduce new method, modify the existing method **installOSFromNIM**

**Review by naveenkandakur — CHANGES_REQUESTED** [review]
Also please consider modifying or adding new pytests when you introduce new parameter.

File: plugins/module_utils/hmc_resource.py (hunk 1 above)
Comment (spoorthys1303) [reply plugins/module_utils/hmc_resource.py]: modified the code using existing method

**Pull request closed without merging**